docs/03-04-function-configuration-file.md

```diff
@@ -0,0 +1,193 @@
+
+See the sample `config.yaml` for an inline Function for which code and dependencies are stored in the Function custom resource (CR) under the **spec.source** and **spec.deps** fields. This specification also contains the definition of a sample Subscription and API Rules for the Function:
+
+```yaml
+name: function-practical-filip5
+namespace: testme
+runtime: nodejs14
+labels:
+    app: serverless-test
+source:
+    sourceType: inline
+    sourcePath: /tmp/cli
+    sourceHandlerName: /code/handler.js
+    depsHandlerName: /dependencies/package.json
+resources:
+    limits:
+      cpu: 1
+      memory: 1Gi
+    requests:
+      cpu: 500m
+      memory: 500Mi
+subscriptions:
+    - name: function-practical-filip5
+      protocol: ""
+      filter:
+        dialect: ""
+        filters:
+            - eventSource:
+                property: source
+                type: exact
+                value: ""
+              eventType:
+                property: type
+                type: exact
+                value: sap.kyma.custom.demo-app.order.created.v1
+apiRules:
+    - name: function-practical-filip5
+      gateway: kyma-gateway.kyma-system.svc.cluster.local
+      service:
+        host: path.34.90.136.181.xip.io
+        port: 80
+      rules:
+        - methods:
+            - GET
+            - POST
+            - PUT
+            - PATCH
+            - DELETE
+            - HEAD
+          accessStrategies: []
+        - path: /path1/something1
+          methods:
+            - PUT
+            - PATCH
+            - DELETE
+          accessStrategies:
+            - handler: noop
+        - path: /path1/something2
+          methods:
+            - GET
+          accessStrategies:
+            - config:
+                required_scope: ["read"]
+              handler: oauth2_introspection
+        - path: /path2
+          methods:
+            - DELETE
+          accessStrategies:
+              handler: jwt
+            - config:
+                jwksUrls:
+                    - http://dex-service.kyma-system.svc.cluster.local:5556/keys
+                    - http://dex-service.kyma-system.svc.cluster.local:5556
+                trustedIssuers:
+                    - https://dex.34.90.136.181.xip.io
+                    - https://dex.34.90.136.181.xip.io
+env:
+    - name: REDIS_PASS
+      value: YgJUg8z6eA
+    - name: REDIS_PORT
+      value: "6379"
+    - name: REDIS_HOST
+      value: hb-redis-enterp-6541066a-edbc-422f-8bef-fafca0befea8-redis.testme.svc.cluster.local
+    - valueFrom:
+        configMapKeyRef:
+          Name: configmap1
+          Key: token-field
+    - valueFrom:
+        secretKeyRef:
+          Name: secret1
+          Key: token-field
+```
+
+## Specification for a Git Function
+
+See the sample `config.yaml` for a [Git Function](https://kyma-project.io/docs/main/components/serverless/#details-git-source-type) for which code and dependencies are stored in a selected Git repository.
```

Comment:
```suggestion
See the sample `config.yaml` for a [Git Function](https://kyma-project.io/docs/main/components/serverless/#details-git-source-type) for which code and dependencies are stored in a selected Git repository: 
```

The first `.yaml` description ended with a `:` so it would be nice to add it here as well for consistency.